app/actions.py

```diff
@@ -62,6 +62,76 @@ def __init__(self, program):
         self.debugUpperChangedRow = -1
         self.parser = app.parser.Parser()
         self.fileFilter(u'')
+    
+    def getMatchingBracketAddress(self):
+        """Returns the address of the bracket which matches
+        the bracket at the current position of the cursor.
+
+        Args:
+          None.
+
+        Returns:
+          None if matching bracket isn't found.
+          Address (int row, int col) of the matching bracket otherwise.
+        """
+        if (self.parser.rowCount() > self.penRow and
+                len(self.parser.rowText(self.penRow)) > self.penCol):
```

Comment:
Unless there's a reason not to: 
Try to reverse the logic in lines 77 and 78, then do `return None` as the body of the `if`; and unindent the rest of the function.
(There are two competing schools of thought here: one is to minimize the number of `return` statements and the other is to early out and keep the normal flow more like a list (keep code to the left column) than nesting code (indenting to the right). I believe the latter is a better choice.)

Reply:
Fixed